04/src/main/scala/ru/otus/sc/user/model/FindUsers.scala

```diff
@@ -3,6 +3,8 @@ package ru.otus.sc.user.model
 sealed trait FindUsersRequest
 object FindUsersRequest {
   case class ByLastName(lastName: String) extends FindUsersRequest
+  case class ByRole(role: Role)           extends FindUsersRequest
+  case class All()                        extends FindUsersRequest
```

Comment:
case class без параметров это case object
```suggestion
  case object All                         extends FindUsersRequest
```